tests/test_prerequisites.py

```diff
@@ -0,0 +1,272 @@
+import unittest
+from unittest import mock
+
+from pythonforandroid.prerequisites import (
+    JDKPrerequisite,
+    HomebrewPrerequisite,
+    OpenSSLPrerequisite,
+    AutoconfPrerequisite,
+    AutomakePrerequisite,
+    LibtoolPrerequisite,
+    PkgConfigPrerequisite,
+    CmakePrerequisite,
+    get_required_prerequisites,
+)
+
+
+class PrerequisiteSetUpBaseClass(object):
```

Comment:
Object is already inherited in Python 3
```suggestion
class PrerequisiteSetUpBaseClass:
```

Reply:
Agree. Old habits are hard to forget ;)